code/style.css

```diff
@@ -2,6 +2,236 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+/* .active + .answer {
+  /* some styling here 
+ */
+
+ * {
+  margin: 0;
+}
+
+body {
+  background: #fff;
+  margin:0; 
+}
+
+/**** HEADER ****/
+
+header {
+background-color: #fff;
+text-align: center;
+
+}
+
+header img {
+  height: 100px;
+  padding: 10px;  
+}
+
+.hero-image {
+  height: 380px;
+  background-image: url('balloons.jpg');
```

Comment:
For the future, try to keep you images in a separate image folder - this will make the file structure better and it's also easier for yourself to navigate through your project.